### DISCUSSION TITLE
Wechselrichter startet Morgens kurz gegen 6Uhr aus unklarem Grund.

### DISCUSSION THREAD

**Discussion by peff74** [hoylabs/OpenDTU-OnBattery#906] [General]
Moin,
in den letzten Tagen ist mir aufgefallen, dass der Wechselrichter Morgens gegen 6Uhr ein paar Mal startet und dann wieder ausgeht.
Obwohl, Laderegler nichts liefert und Batterie natürlich nicht über dem Start Sollwert liegt.
Ich nutze z.Z. die 
https://github.com/helgeerbe/OpenDTU-OnBattery/commits/1d4bea2
aus dem Development.

Habe ich eine neue Einstellung übersehen, die das hervorruft, oder ist's nen Bug?

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/cd1aa42f-6ad3-49ee-9fd8-472e1e01e86a)
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/10289aff-2504-41d6-82cf-9d7268cdf3dc)

**Comment by schlimmchen** [comment]
Hä? Was ist denn da los? :astonished: 

Hast du einen zweiten Laderegler im VE.Smart Netzwerk, der vielleicht stattdessen Leistung liefert, die den DPL veranlassen könnte Solar-Passthrough zu starten?

Die Start-Spannung für die Batterie ist weit weg von 40V?

**Comment by peff74** [comment]
Ja, das Frage ich mich auch.
Heute Morgen das Gleiche

hier mal der ganze Tag.
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/a2647f71-417f-4393-8680-249918d86735)

Und hier im Detail.
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/f804696a-d084-4930-b76b-ee97d9a5cd24)

Man sieht deutlich das der Akku gegen 05:35 leer ist.
Aber gegen 05:53 startet der WR kurz, mehrfach unklar warum.
Der Victron erwacht gegen 5:57 zum Leben.
Einschaltspannung wäre ja erst 43.5V

Aufbau ist simpel, nur ein Hoymiles und ein Victron.



Hier noch ein paar Details.
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/d66e234a-3ee9-48fa-8982-141371ad9084)
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/4a26ccaf-31c1-492d-897a-a2d4d098b445)

**Reply by SW-Niko** [reply]
Hallo @peff74 ,
Welche Spannungs-Messquelle liefert bei dir die Werte für die Abschaltung? Inverter, MPPT oder BMS?
Und dann fällt mir noch auf, das die Batterie erheblich unter die 40VDC entladen wird. 
Ein Lastkorrekturfaktor von 0.006 bei 40VDC ergibt ja einen Widerstand von ca. 260mOhm. Was ist den bei dir alles zwischen Batterie und Inverter um auf einen so hohen Wert zu kommen?

**Reply by peff74** [reply]
Hi,
bisher, habe gerade gesehen, dass es auch anders geht^^, liefert der WR.
Siehste ja an dem Shot meiner Einstellungen.
Und natürlich wird die Batt unter 40V entladen. Das hat ja nicht mehr Widerstand zwischen BATT und Inverter zu tun. Sondern damit, dass die Spannung der Akkus einbricht bei Last. 
Bei 0,006 Lastkorrektur lande ich dann immer ziemlich genau bei ~40V Leerspannung bei S12 denke ich O.K.

Ich habe ~0.8V Spannungsabfall zwischen LR und BMS bei 20A was auf ~0.04Ohm kommt.
Glaube, das liegt im Rahmen.

**Reply by SW-Niko** [reply]
Hallo,

> liefert der WR. Siehste ja an dem Shot meiner Einstellungen.

Nein sorry, das sehe ich nicht aus den Einstellungen. Die OpenDTU nimmt von den verfügbaren Werten automatisch den besseren.
Ist das BMS oder der MPPT an die OpenDTU angebunden?

> Ich habe ~0.8V Spannungsabfall zwischen LR und BMS bei 20A was auf ~0.04Ohm kommt.

Das ist Ok. Dann hat deine Batterie ja ~0,22Ohm. Was hast du für eine wenn ich fragen darf?

**Reply by peff74** [reply]
das ist nen LiIon Eigenbau aus 12 x 40 18650gern.
MPPT is mit an der OpenDTU
BMS ist über Bluetooth, aber nun auch über MQTT im OpenDTU mit dran.
Und ich bin begeistert, dass die Werte genutzt werden ;-)

**Reply by SW-Niko** [reply]
Interessant....

Zurück zu deinem Problem. Wenn das Logging für den DPL aktivierst dann bekommst du eine Info über die verwendete Messquelle.
19:09:26.131 > [DPL::getBatteryVoltage] BMS: -1.00 V, MPPT: 26.23 V, inverter: 25.90 V, returning: 26.23V
Bei mir kommen die Werte vom MPPT oder noch genauer von der "Smart Battery Sense" die an den MPPT gekoppelt ist.

Falls bei dir das BMS verwendet wird dann bin ich raus. Mit Mqtt kenne ich mich nicht aus.

**Reply by peff74** [reply]
Ja Du hast Recht, nun werden die Werte vom BMS genutzt
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/6a7c7c78-1fbb-4362-b9cc-62eb46393d71)

Trotzdem war es heute Morgen wieder.
Ich sehe es halt nur, wenn der Akku am Vortag leergelaufen war...
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/d861b137-fdb2-4795-8199-b85cc4f967a4)


Gestern Abend hatte ich dann plötzlich noch ein merkwürdiges Phänomen:
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/18d47484-7ec2-4cd3-a5b9-16f68cc7a522)

Hier stellte sich dann aber diese neue Funktion als Ursache heraus:
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/0581e831-1333-42b6-b722-ead791a652e8)
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/ef056136-3a63-47ed-a817-8d30a8e8f120)
Habe es auf 10 gestellt, dann war Ruhe.
Mein WR scheint einer von der trägen Sorte zu sein^^

**Reply by SW-Niko** [reply]
Ok... Das kenne ich noch nicht. 😄 
Wieder was gelernt.

**Reply by peff74** [reply]
ich habe jetzt mal beim ALI nen OpenLog bestellt, den klemm ich dann mal an der OpenDTU und schaue was da um 6Uhr passiert...
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/9ff6eb78-5c0a-4602-b380-b47e9869e417)

**Comment by peff74** [comment]
was auch immer es ist...
Es erscheint, wenn die erste Energie über den LR kommt.
Es rutscht weiter in die früh.
Hat das denn sonst keiner hier?
Man sieht es halt nur, wenn der Akku in der Nacht durch den DPL abgeschaltet wurde...

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/d6aea31d-eec2-4b5f-9b89-a2ec29929c6a)
Hoffe der openLog kommt bald und bringt Klarheit

**Reply by SW-Niko** [reply]
Hast du dir in der Victron App schon mal den Zeitverlauf der Batteriespannung angeschaut?
